build.gradle

```diff
@@ -104,6 +105,43 @@ switch (JavaVersion.current()) {
         break;
 }
 
+task cloneAndBuildJspecify {
+    description 'Obtain or update JSpecify.'
+    doLast {
+        if (JavaVersion.current() > JavaVersion.VERSION_1_8) {
```

Comment:
It's clearer if you exit early if the condition isn't met. Add a comment why.

Reply:
already add the comment.